corehq/apps/app_manager/static/app_manager/spec/form_workflow_spec.js

```diff
@@ -1,36 +1,40 @@
+import _ from "underscore";
+import sinon from "sinon/pkg/sinon";
+
+import Toggles from "hqwebapp/js/toggles";
+import FormWorkflow from "app_manager/js/forms/form_workflow";
```

Comment:
is it possible to do an import statement like
`import {FormWorkflow} from "app_manager/js/forms/form_workflow";` so that we avoid the `FormWorkflow.FormWorflow` situation below?

Reply:
hmm nvm perhaps that isn't possible because it's not importing from an ESM module

Reply:
I'd need to update the return value of `app_manager/js/forms/form_workflow` and therefore the other code that depends on that module. I might get there - I don't like `FormWorkflow. FormWorkflow` either - but not in this PR.

Reply:
sounds good to me!